superset/assets/visualizations/chord.jsx

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-param-reassign */
+import d3 from 'd3';
+import { category21 } from '../javascripts/modules/colors';
+import './chord.css';
+
+function chordViz(slice, json) {
+  slice.container.html('');
+
+  const div = d3.select(slice.selector);
+  const nodes = json.data.nodes;
+  const fd = slice.formData;
```

Comment:
On of the things that the slice object exposes isf the information provided by the user controls. The keys of this `slice.formData` object are the control's "id" defined in `controls.jsx` and their values are the ones provided by the user.